cmd/kogito/command/project/project.go

```diff
@@ -39,27 +36,32 @@ func addProjectFlagsToCommand(command *cobra.Command, pFlags *projectFlags) {
 	command.Flags().BoolVar(&pFlags.installDataIndex, "install-data-index", false, message.InstallDataIndex)
 	command.Flags().BoolVar(&pFlags.installJobsService, "install-jobs-service", false, message.InstallJobsService)
 	command.Flags().BoolVar(&pFlags.installManagementConsole, "install-mgmt-console", false, message.InstallMgmtConsole)
-	command.Flags().BoolVar(&pFlags.installAll, "install-all", false, message.InstallAllServices)
-	command.Flags().BoolVar(&pFlags.enablePersistence, "enable-persistence", false, "If set will install Infinispan in the same namespace and inject the environment variables to configure the service connection to the Infinispan server.")
-	command.Flags().BoolVar(&pFlags.enableEvents, "enable-events", false, "If set will install a Kafka cluster via the Strimzi Operator. ")
 }
 
 func handleServicesInstallation(pFlags *projectFlags, cli *client.Client) error {
 	install := shared.
 		ServicesInstallationBuilder(cli, pFlags.project).
 		SilentlyInstallOperatorIfNotExists(shared.KogitoChannelType(pFlags.Channel))
 
-	if pFlags.installAll || pFlags.installDataIndex {
+	if pFlags.installDataIndex || pFlags.installJobsService || pFlags.installManagementConsole {
```

Comment:
Please drop Jobs service and Management console from here too.

Reply:
Removed